.vscode/tasks.json

```diff
@@ -44,6 +44,12 @@
 				"reveal": "never"
 			},
 			"group": "build"
+		},
+		{
+			"label": "Parse quarry",
+			"type": "shell",
+			"command": "npx tree-sitter query scopeTypes.scm ts-test.ts",
```

Comment:
Couldn't get this one to work but will prob be useful to have a task or launch config to use while hacking.  Maybe `launch.json` is better